src/include/distributed/worker_manager.h

```diff
@@ -44,6 +46,7 @@ typedef struct WorkerNode
 	bool hasMetadata;                   /* node gets metadata changes */
 	bool isActive;                      /* node's state */
 	Oid nodeRole;                       /* the node's role in its group */
+	char nodeCluster[WORKER_LENGTH];    /* the cluster the node is a part of */
```

Comment:
Might make sense to use NAMEDATALEN here, just in case we ever get crazy and want to create some table, schema, or role with the same name.